auth/local/index.js

```diff
@@ -1,13 +1,16 @@
 const { Router } = require('express');
+const { isAuthenticated } = require('../auth.service');
 
 const {
   loginUserHandler,
   verifyAccountHandler,
+  changePassword,
 } = require('./auth.controller');
 
 const router = Router();
 
 router.post('/login', loginUserHandler);
 router.get('/verifyAccount/:token', verifyAccountHandler);
+router.patch('/:id', isAuthenticated, changePassword);
```

Comment:
It would make more sense if the path was:

```suggestion
router.patch('/change-password', isAuthenticated, changePassword);
```